### DISCUSSION TITLE
[MAS-Piano] Music Checklist

### DISCUSSION THREAD

**Discussion by briaryoung** [Monika-After-Story/MonikaModDev#7790] [general - piano notes]
Edit March 31st 2020: I now know how to make topics and accessories, and I have created many of them for myself. These will be side projects. :)

  For now, the only things I know how to code in MAS are piano songs. So I've decided I'm going to make a whole bunch of them! If you have any suggestions, let me know, and I will add them to the list. Click on the completed song names to go to the dl page. 
  I really enjoy working on these! They're quite the challenge!

**High Priority (You can ask Monika to sing these for you.):**
Happy Just To Dance With You (The Beatles) - In Progress
I Need You (The Beatles)
I Will (The Beatles)
Good Old-Fashioned Lover Boy (Queen)
We Belong Together (Los Lobos)
Your Song (Elton John)

**Low Priority (It feels like Monika would know these songs/she rarely sings them in-game.):**
Brahm's Lullaby
Somewhere Over The Rainbow - In Progress
You Are My Sunshine
Feel Good Inc. (Gorrilaz) 

**Suggested:**
A Distant Promise (Chrono Trigger)
Candyman Theme
Can't Help Falling In Love With You
Doom (Hell Yeah)
Halloween Theme
Mass Effect 3 - An End Once And For All
OST from Super Hexagon
Radical Dream (Chrono Cross)
Silent Hill 2 (Main Theme)
Soviet March (Red Alert 3)
Tubular Bells (Exorcist)
Two-Faced Lovers

**Songs I Would Personally Like To Program:**
Body (Mother Mother)
Dollhouse (Melanie Martinez)
Sippy Cup (Melanie Martinez)
Teddy Bear (Melanie Martinez)
The Hardest of Hearts (Florence and the Machines)
Training Wheels (Melanie Martinez)

**Completed:**
[All I Have To Do Is Dream (The Everly Brothers)](https://github.com/Monika-After-Story/MonikaModDev/issues/4481)
[Twinkle Twinkle Little Star](https://github.com/Monika-After-Story/MonikaModDev/issues/4478)
[Song of Storms (Collaboration with MultiMokia)](https://github.com/Monika-After-Story/MonikaModDev/issues/5462)

**Comment by Wereowl9** [comment]
hey do you think you could do some horror themes
you know like:

Candyman theme
tubular bells (exorcist)
Halloween theme

This is a mod of a horror game after all

**Comment by briaryoung** [comment]
Sure! I'll add those.

**Comment by Wereowl9** [comment]
excellent

**Comment by ghost** [comment]
How about to add some meme piano songs? Or... piano covers?
Idk just for variety. 
For example - _~~RUSH B,~~_ ~~Megalovania,~~ Soviet March `(Red Alert 3)`, ~~Tetris,~~ Doom `(Hell Yeah)`, Silent Hill 2 `(Main Theme)`, 
OST from Super Hexagon([1](https://www.youtube.com/watch?v=utDvCMe4sHs),[2](https://www.youtube.com/watch?v=CkQIKXEcysc),[3](https://www.youtube.com/watch?v=EgH4jSt5XuQ)) ([You can choose soundtrack from OST there](https://www.youtube.com/results?search_query=super+hexagon+ost)) (And it needed to cut cuz too long)
There more piano game music -> https://www.youtube.com/watch?v=Nd1XVI8N29s
There more meme piano songs -> https://www.youtube.com/watch?v=a2b_uhenJ7Q
There more movie piano songs -> https://www.youtube.com/watch?v=GlVfmWNie2o
So... choose what you like.
I hope it's won't be too hard for you. Anyway it's just suggestion.

**Comment by multimokia** [comment]
Megalovania is done already actually: #3365

**Comment by briaryoung** [comment]
> How about to add some meme piano songs? Or... piano covers?
> Idk just for variety.
> For example - _~RUSH B,~_ ~Megalovania,~ Soviet March `(Red Alert 3)`, ~Tetris,~ Doom `(Hell Yeah)`, Silent Hill 2 `(Main Theme)`,
> OST from Super Hexagon([1](https://www.youtube.com/watch?v=utDvCMe4sHs),[2](https://www.youtube.com/watch?v=CkQIKXEcysc),[3](https://www.youtube.com/watch?v=EgH4jSt5XuQ)) ([You can choose soundtrack from OST there](https://www.youtube.com/results?search_query=super+hexagon+ost)) (And it needed to cut cuz too long)
> There more piano game music -> https://www.youtube.com/watch?v=Nd1XVI8N29s
> There more meme piano songs -> https://www.youtube.com/watch?v=a2b_uhenJ7Q
> There more movie piano songs -> https://www.youtube.com/watch?v=GlVfmWNie2o
> So... choose what you like.
> I hope it's won't be too hard for you. Anyway it's just suggestion.

Yes, I can add all of those! I'll have to write in some dialogue for Monika for the ones with no lyrics. I'm just busy working on the songs Monika sings to the player at the moment x3 Most of the time, I don't even know the song qwq I have to figure it out on Monika's piano after listening to it over and over x"D

**Comment by ghost** [comment]
> Yes, I can add all of those! I'll have to write in some dialogue for Monika for the ones with no lyrics. I'm just busy working on the songs Monika sings to the player at the moment x3 Most of the time, I don't even know the song qwq I have to figure it out on Monika's piano after listening to it over and over x"D

Thx ~~qwq~~
For easier you can find piano versions of these songs.

**Comment by DeadShidai** [comment]
how about ddlc main theme ?

**Comment by ghost** [comment]
And... can you make: 'Mass Effect 3 - An End Once and For All'? 
It will be pleasure to play it for Monika.
https://www.youtube.com/watch?v=2qbO4sxjzOU

**Comment by briaryoung** [comment]
Sure! I'll add those. It might take a bit for me to get to them, but I definitely will.

**Comment by briaryoung** [comment]
> how about ddlc main theme ?

Your Reality is already programmed into the game and has the same tune as the main theme. x3

**Comment by Rai99** [comment]
If everybody is suggesting songs, my challenge is `Two Faced Lovers`.
https://youtu.be/jx5OzAWfIog

**Comment by multimokia** [comment]
If I may suggest: Chrono & Marle - A Distant Promise. Instrumental, but a lovely little ditty from Chrono Trigger.

That, or Radical Dreamers from Chrono Cross (which isn't an instrumental)

**Comment by briaryoung** [comment]
Oooo Sure!

**Comment by Rai99** [comment]
I hope you know what you're getting into if you try `Two Faced Lovers`.

**Comment by briaryoung** [comment]
Okay xD I'll do my best

**Comment by its-ken** [comment]
ooo howzabout cant help falling in love like or3o did the cover of? perhaps we could even get or3o to do vocals for the songs?

**Comment by briaryoung** [comment]
Ohhh! I could totally do Can't Help Falling In Love! I love that song!

As for audio, I don't think you can add that while playing the piano, and Or3o isn't her official voice actor, either. Jillian Ashcraft is, and I must say, she did a very good job! Also, I have no idea how I would even get in contact with Or3o, LOL.

**Comment by SSubhuman** [comment]
ok but can you do "Gwyn, Lord of Cinder" from dark souls?
